code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -302,9 +302,10 @@
 // Jetpack things
 // Please kill me
 
+//called in datum/component/jetpack/proc/activate() : (source, user)
+#define JETPACK_COMPONENT_ACTIVATION_FAILED (1<<0)
```

Comment:
what are you trying to do here exactly? it's still used t he same why was it touched

Reply:
Why have different returns values? If it fails let it just return one value that can be understood by everyone. 
Yes the previous one worked but it was just execess semantics sugar

Reply:
After all you just acknowledged this


> 5. Only 1 return value from the jetpack component when activation fails rather than passing a unique value for every case for consistency
> Ok sure

Reply:
called isn't the right value here. though actually, I think I no longer agree with 2 hours ago me.
we want other consumers of these signals to be able to block them. They don't have them right now but we should support it, and I think that means we want a unique flag for each, if at least by name.

Reply:
This define isn't tied to any particular signal, it's just a general failure for anything that tries to activate the jetpack component via any signal because if you look at the Initialize() proc you can see it is passed as a param `activation_signal`
https://github.com/tgstation/tgstation/blob/583538a12afcb5ef6e9a3b5e04d743b032569f3d/code/datums/components/jetpack.dm#L27

Which registers itself on the `activate` proc
https://github.com/tgstation/tgstation/blob/583538a12afcb5ef6e9a3b5e04d743b032569f3d/code/datums/components/jetpack.dm#L34

Which returns a failure if it occurs
https://github.com/tgstation/tgstation/blob/583538a12afcb5ef6e9a3b5e04d743b032569f3d/code/datums/components/jetpack.dm#L51-L52

So any future thing can pass in whatever signal via `activation_signal` and listen for the failure as follows
```
if(SEND_SIGNAL(src, <WHATEVER_SIGNAL>, user) & JETPACK_COMPONENT_ACTIVATION_FAILED)
   return //we have nothing to do here
```


So here the signal can be unique & whatever, & having a universal return value for failed activation creates a standard for eveyone to understand